src/uwtools/api/driver.py

```diff
@@ -91,25 +100,6 @@ def tasks(module: str, classname: str) -> dict[str, str]:
     return _tasks(class_)
 
 
-_CLASSNAMES = [
-    "Assets",
-    "AssetsCycleBased",
-    "AssetsCycleLeadtimeBased",
-    "AssetsTimeInvariant",
-    "Driver",
-    "DriverCycleBased",
-    "DriverCycleLeadtimeBased",
-    "DriverTimeInvariant",
-]
-
-
-def _add_classes():
-    m = import_module("uwtools.drivers.driver")
-    for classname in _CLASSNAMES:
-        setattr(sys.modules[__name__], classname, getattr(m, classname))
-        __all__.append(classname)
```

Comment:
The dynamic addition of the driver classes to the module is no longer needed since they're being explicitly imported now.